### DISCUSSION TITLE
support for nuxt3

### DISCUSSION THREAD

**Discussion by andysay** [nuxt-community/sitemap-module#234] [Q&A] [answered]
when is nuxt3 support planned ?

**Comment by benoitdemaegdt** [comment, marked as answer]
Hi there 👋 

While waiting for nuxt3 support of the sitemap-module, you can follow these steps for generating a simple sitemap for your app :
https://github.com/benoitdemaegdt/nuxt3-sitemap

I hope this can help some of you, even if it is minimalistic 😃

**Comment by thisismydesign** [comment]
I made a workaround for Nuxt 2 Bridge, might work for Nuxt 3 as well: https://github.com/nuxt-community/sitemap-module/issues/281
